src/util/clickTracking.js

```diff
@@ -134,6 +134,7 @@ function getTrackClick(store) {
 
 		// 2. Create click action with metadata
 		const clickTrackAction = click({
+			timestamp: new Date().toISOString(),
```

Comment:
part of me thinks it's nice to abstract this into a `getISOString()` function since we're doing it more than once... but, it's also only one line.

Might be nice to make them consistent, though, and either set an eventDate variable or not (I also like the syntax with an extra set of parens like `(new Date()).toISOString()` )